tidb-configuration-file.md

```diff
@@ -569,16 +569,16 @@ Configuration related to the status of TiDB service.
 
 ## stmt-summary <span class="version-mark">New in v3.0.4</span>
 
-Configurations related to the `events_statement_summary_by_digest` table.
+Configurations related to [statement summary tables](/statement-summary-tables.md).
 
 ### max-stmt-count
 
-- The maximum number of SQL categories allowed to be saved in the `events_statement_summary_by_digest` table.
-- Default value: `100`
+- The maximum number of SQL categories allowed to be saved in [statement summary tables](/statement-summary-tables.md).
+- Default value: `200`
```

Comment:
```suggestion
- In versions earlier than v4.0.14, the default value of this variable is `200`. Since v4.0.14, the default value of this variable is `3000`
```